Dockerfile

```diff
@@ -0,0 +1,68 @@
+# Pull from the latest Ruby version
+# We don't use slim or alpine variants as we invariably need build tooling, which
+# the base image includes for us.
+FROM ruby
+
+# Avoid issues with file encoding in Ruby by setting these two environment variables
+# This tells the underlying OS to expect UTF-8 encoding (e.g. UTF-8 encoding in the US language)
+ENV LANG "C"
+ENV LC_ALL C.UTF-8
+
+
+# Curl is installed to make it possible to set up PPAs below - it is
+# removed further down.
+RUN apt-get update -qq &&\
+    apt-get install -y curl 
+
+# Install Node.js PPA for asset management
+# As of writing, Node 10 is the most recent LTS.
```

Comment:
🐘 I'd be happy for this to be bumped to `12`, since that enters LTS in a month or so, and we typically don't add `engines` to our `package.json` meaning our CIs typically run against `12` anyway.

I'd rather know if something can't work against `12` asap than after we've built the project and we're trying to bump the node version years later (a few projects in that situation spring to mind, as well as gulp@3).

Reply:
@G-Rath I think we'd handle that at the time the LTS is officially tagged in a PR to this repo

Reply:
Sure; I just know who'll be assigned the CC task of running around bumping the node version of all of our projects 😉